chalk-solve/Cargo.toml

```diff
@@ -27,5 +26,5 @@ chalk-integration = { path = "../chalk-integration" }
 [features]
 default = ["slg-solver", "recursive-solver"]
 
-slg-solver = ["chalk-engine"]
+slg-solver = []
 recursive-solver = []
```

Comment:
Can we remove these now?

Reply:
Seconding this, from my review of the previous PR: https://github.com/rust-lang/chalk/pull/542#discussion_r446659371

Reply:
@Mcat12 sorry, can you elaborate on this. What do we ideally want to do here

Reply:
We should be able to remove the slg and recursive feature flags.